test/testdata/simpletests/datavolume2.json

```diff
@@ -0,0 +1,11 @@
+{
+  "Name": "DataVolume2",
+  "Description": "Verify that more complex datavolumes (including empty and volumes-from) work as expected; see Related",
+  "TaskDefinition": "datavolume2",
+  "Version": ">1.0.0",
+  "Timeout": "2m",
+  "ExitCodes": {
+    "exit": 42
+  },
+  "Related": ["forums#176057", "gh#52"]
```

Comment:
I'll expand it into proper English and links. I think I've spent too much time in the company of those irc bots that expand gh#xyz for you :smile: